### PR TITLE
[Server] Fix operation and schema URL for WFS GetFeature response

### PR DESCRIPTION
**Pull request by pathmapper**
## Description

This PR fixes two unreported bugs for the QGIS Server WFS GetFeature response:

1. Operation `DescribeFeatureType` -> `GetFeature`
2. Schema URL for WFS 1.1.0: 
`http://schemas.opengis.net/wfs/1.0.0/wfs.xsd` -> `http://schemas.opengis.net/wfs/1.1.0/wfs.xsd`

image

### CODE REVIEW

File: src/server/services/wfs/qgswfsgetfeature.cpp [hunk 1]
```diff
@@ -1054,7 +1054,7 @@ namespace QgsWfs
             query.removeAllQueryItems( param.first );
         }
 
-        query.addQueryItem( QStringLiteral( "REQUEST" ), QStringLiteral( "DescribeFeatureType" ) );
+        query.addQueryItem( QStringLiteral( "REQUEST" ), QStringLiteral( "GetFeature" ) );
```
Comment (rouault) [inline src/server/services/wfs/qgswfsgetfeature.cpp:1057]: this change is not correct. The xsi:schemaLocation of a GetFeature response should point to a DescribeFeatureType request to get the schema

File: src/server/services/wfs/qgswfsgetfeature.cpp [hunk 2]
```diff
@@ -1156,7 +1162,7 @@ namespace QgsWfs
...
```
Comment (rouault) [inline src/server/services/wfs/qgswfsgetfeature.cpp:1165]: this change is not correct. The xsi:schemaLocation of a GetFeature response should point to a DescribeFeatureType request to get the schema

**Review by rouault — CHANGES_REQUESTED** [review]

File: src/server/services/wfs/qgswfsgetfeature.cpp (hunk 1 above)
Comment (pathmapper) [reply src/server/services/wfs/qgswfsgetfeature.cpp:1057]: My bad, thanks @rouault - fresh PR in https://github.com/qgis/QGIS/pull/45138

**Comment by pathmapper** [comment]
fresh PR in #45138

**Pull request closed without merging**